rust/stackable-cockpit/src/platform/stack/spec.rs

```diff
@@ -159,15 +159,9 @@ impl StackSpecV2 {
         release_list: ReleaseList,
         operator_namespace: &str,
         product_namespace: &str,
-        skip_release_install: bool,
     ) -> Result<(), StackError> {
         info!("Installing release for stack");
```

Comment:
Please remove the "for stack" (you can only install a release with no stack)" and mention the release name

Reply:
Changed in baffb30